microprofile/tests/tck/tck-core-profile/tck-core-profile-test/pom.xml

```diff
@@ -30,6 +30,11 @@
     <name>Helidon Microprofile Tests TCK Core Profile</name>
 
     <dependencies>
+        <dependency>
```

Comment:
why do you depend on the whole thing here? It should be sufficient to depend on the core, as core profile should be already supported by core.

Reply:
Right, for some reason I thought this was testing a small parts of different microprofile things.